site/content/en/docs/handbook/registry.md

```diff
@@ -44,7 +44,7 @@ One nifty hack is to allow the kubelet running in minikube to talk to registries
 with TLS certificates. Because the default service cluster IP is known to be available at 10.0.0.1, users can pull images from registries
 deployed inside the cluster by creating the cluster with `minikube start --insecure-registry "10.0.0.0/24"`.
 
-### docker on macOS
+### docker on Linux/macOS
```

Comment:
I don think this is correct, docke ron linux does Not need port forwarding.
and docker on mac os needs that.

Reply:
Hey @medyagh thanks for the review, I've addressed your comment.

Reply:
this is not correct. docker on linux is different

Reply:
I admit this is confusing for most users, this is because docker on mac is differnt on docker on linux we could probably find a bewtter way to explaion this but I will have to close this pr